### PR TITLE
ENG-2814: Should valid passwordConfirm on user creation

### PR DESCRIPTION
**Pull request by werbth**
…confirm password do not match

### CODE REVIEW

File: src/main/java/org/entando/entando/web/user/model/UserRequest.java [hunk 1]
```diff
@@ -32,6 +32,8 @@
     //@Size(min = 6, message = "user.password.size")
     private String password;
 
+    private String passwordConfirm;
```
Comment (cecchisandrone) [inline src/main/java/org/entando/entando/web/user/model/UserRequest.java:35]: better `passwordConfirmation`

File: src/main/java/org/entando/entando/web/user/validator/UserValidator.java [hunk 2]
```diff
@@ -76,6 +76,8 @@
 
     public static final String ERRCODE_UPDATE_OTHER = "9";
 
+    public static final String ERRCODE_PASSWORD_CONFIRM_MISMATCH = "10";
```
Comment (cecchisandrone) [inline src/main/java/org/entando/entando/web/user/validator/UserValidator.java:79]: same as above

File: src/main/java/org/entando/entando/web/user/model/UserRequest.java (hunk 1 above)
Comment (werbth) [reply src/main/java/org/entando/entando/web/user/model/UserRequest.java:35]: I am matching the parameter name that I receive from the UI. This is also specified in the acceptance criteria in the ticket.

File: src/main/java/org/entando/entando/web/user/validator/UserValidator.java (hunk 2 above)
Comment (werbth) [reply src/main/java/org/entando/entando/web/user/validator/UserValidator.java:79]: Please check the comment above.

File: src/main/java/org/entando/entando/web/user/model/UserRequest.java (hunk 1 above)
Comment (cecchisandrone) [reply src/main/java/org/entando/entando/web/user/model/UserRequest.java:35]: mmm ok

**Review by BrenoQVDS — APPROVED** [review]

**Pull request merged**